zipline/utils/calendars/us_futures_calendar.py

```diff
@@ -63,6 +69,12 @@ def close_time(self):
     def open_offset(self):
         return -1
 
+    def execution_time_from_open(self, open_date):
+        return open_date + Timedelta(hours=FUTURES_OPEN_TIME_OFFSET)
+
+    def execution_time_from_close(self, close_date):
+        return close_date + Timedelta(hours=FUTURES_CLOSE_TIME_OFFSET)
```

Comment:
@richafrank Regarding your performance question, so should I just have these methods return a Timedelta and then do the addition in `algorithm.py`?